### PR TITLE
Revert wchar / rune mapping

### PR DESCRIPTION
**Pull request by ptxmac**
This reverts the changes made to map wchar to rune

See #149

### CODE REVIEW

File: cmd/codegen/return_wrapper.go [hunk 1]
```diff
@@ -13,9 +13,8 @@ func getReturnTypeWrapperFunc(returnType string) (returnWrapper, error) {
 		"bool":                     {"bool", "return %s == C.bool(true)"},
 		"char*":                    {"string", "return C.GoString(%s)"},
 		"const char*":              {"string", "return C.GoString(%s)"},
-		"const ImWchar*":           {"*rune", "return (*rune)(unsafe.Pointer((*uint)(unsafe.Pointer(%s))))"},
-		"ImWchar":                  simpleR("rune"),
-		"ImWchar16":                simpleR("rune"),
+		"const ImWchar*":           simpleR("(*Wchar)"),
+		"ImWchar":                  simpleR("Wchar"),
```
Comment (gucio321) [inline cmd/codegen/return_wrapper.go:17]: maybe leave Wchar16 as `uint16`? It should work I think...

File: cmd/codegen/arguments_wrapper.go [hunk 2]
```diff
@@ -24,8 +24,6 @@ type argumentWrapper func(arg ArgDef) ArgumentWrapperData
 
 func argWrapper(argType string) (wrapper argumentWrapper, err error) {
 	argWrapperMap := map[string]argumentWrapper{
-		"char":                     simpleW("rune", "C.char"),
-		"char[5]":                  simplePtrArrayW(5, "C.char", "rune"),
```
Comment (gucio321) [inline cmd/codegen/arguments_wrapper.go:28]: according to my [google search results](https://www.google.com/search?q=what+is+bit+size+of+char+type+in+C) it seems that ith should be OK - the table says that char size is 32 so it should be the same as go's rune

image

**Review by gucio321 — CHANGES_REQUESTED** [review]
ok, I see the problem but I think we can minimize losses

**Comment by ptxmac** [comment]
You're right, I reverted a little too much 😄 
Updated it to keep `char` and wchar16

**Review by gucio321 — APPROVED** [review]
ok, thank you!

**Pull request merged**